Source/Combine.swift

```diff
@@ -571,17 +571,17 @@ extension DownloadRequest {
     /// response.
     ///
     /// - Parameters:
-    ///   - type:                `Decodable` type to which to decode response `Data`. Inferred from the context by default.
-    ///   - queue:               `DispatchQueue` on which the `DataResponse` will be published. `.main` by default.
-    ///   - preprocessor:        `DataPreprocessor` which filters the `Data` before serialization. `PassthroughPreprocessor()`
-    ///                          by default.
-    ///   - decoder:             `DataDecoder` instance used to decode response `Data`. `JSONDecoder()` by default.
-    ///   - emptyResponseCodes:  `Set<Int>` of HTTP status codes for which empty responses are allowed. `[204, 205]` by
-    ///                          default.
-    ///   - emptyRequestMethods: `Set<HTTPMethod>` of `HTTPMethod`s for which empty responses are allowed, regardless of
-    ///                          status code. `[.head]` by default.
-    ///
-    /// - Returns:               The `DownloadResponsePublisher`.
+    ///   - type:                 `Decodable` type to which to decode response `Data`. Inferred from the context by default.
+    ///   - queue:                `DispatchQueue` on which the `DataResponse` will be published. `.main` by default.
+    ///   - preprocessor:         `DataPreprocessor` which filters the `Data` before serialization.
+    ///                           `PassthroughPreprocessor()` by default.
+    ///   - decoder:              `DataDecoder` instance used to decode response `Data`. `JSONDecoder()` by default.
+    ///   - emptyResponseCodes:   `Set<Int>` of HTTP status codes for which empty responses are allowed. `[204, 205]` by
+    ///                           default.
+    ///   - emptyResponseMethods: `Set<HTTPMethod>` of `HTTPMethod`s for which empty responses are allowed, regardless
```

Comment:
This should actually be `emptyRequestMethods` to match the underlying function call. We can leave it for now or you can create a properly named method and deprecate this one, copying the docs from this method and then removing them. The deprecated method could then call through to the proper method.